src/test/java/uk/gov/hmcts/reform/profilesync/domain/SyncJobConfigTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.profilesync.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class SyncJobConfigTest {
+
+    String firstSearchQuery = "firstsearchquery";
+    String configRun = "1h";
```

Comment:
private final

Reply:
this has been addressed in the following commit: https://github.com/hmcts/rd-profile-sync/pull/181/commits/bc37603637e5fcbbc46bdd8621f47415d846e2c2